docs/deploys.md

```diff
@@ -0,0 +1,120 @@
+# Halyard Deploys
+
+## Purpose
+
+Explain how Halyard handles deploying Spinnaker.
+
+## Terminology
+
+* **Deployment Type**: How Spinnaker is divided among all the instances it is
+ running on. An example is having each Spinnaker service run a separate
+ instance. See [Deployment Types](#deployment-types) section for more
+ examples.
+* **Service Dependencies**: Dependencies of Spinnaker that need to be run as
+ stand-alone services. An example is Redis, a non-example is Packer.
+* **Deployment Environment**: The infrastructure Spinnaker is running on,
+ Spinnaker's Deployment Type, and Spinnaker's service dependencies.
+
+## Deployment Types
+
+* **LocalhostDebian**: This is every Spinnaker service installed as a Debian
+ package on the node Halyard is running on.
+* **LocalhostGit**: This is every Spinnaker service installed from Git on the 
+ node Halyard is running on. This is intended for use by Developers.
+* **Flotilla**: This is every Spinnaker service installed remotely on a
```

Comment:
Is this a new term or one in use? It's cute, but we should agree on the term.

Flotilla is cute, but more standard would be something like "standalone" and "distributed" (StandaloneDebian, StandaloneGit) and Distributed.

If a flotilla adds a spinnaker twist to distributed (e.g. flotillas are deployments managed by halyard) then ok. Otherwise be wary about introducing new concepts.